### PR TITLE
Correct exercise 1.3.5.c

### PR DESCRIPTION
**Pull request by LaMavia**
# Summary of Changes

- Please indicate the sections you are modifying:
  - exercise 1.3.5.b

# Checklist:

Make sure that your contributions:

- [x] Are accurate, state assumptions
- [x] Use good LaTeX practices (hopefully)
- [x] Compiles: I couldn't compile it locally (package issues), but the part I added compiled on overleaf.

Justification for the change:
[property 5](http://mathonline.wikidot.com/deleted:properties-of-the-supremum-and-infimum-of-a-set)
[wikipedia; section: properties](https://en.wikipedia.org/wiki/Infimum_and_supremum)

### CODE REVIEW

**Comment by LaMavia** [comment]
Sorry for the confusion with 1.3.5.b and c; I'm using the 2nd edition where it's only a and b.
While I'm at it, thank you for the repo, it's helped me immensely, so far

File: chapters/chapter1/chapter1-3.tex [hunk 1]
```diff
@@ -93,7 +93,13 @@ \section{The Axiom of Completeness}
 		&\Rightarrow\, \frac{s}{c} = \sup A \\
 		&s = c \sup A
 	\end{align*}
-	\item If $c < 0$, $\sup(cA) = -c\inf(A)$.
+	\item If $c < 0$, \begin{align*}
```
Comment (mikinty) [inline chapters/chapter1/chapter1-3.tex]: I'm being pretty picky, but can you make the align start on a new line?

**Comment by mikinty** [comment]
Also do you mind putting this into one commit? I think it's a bit messy to have 5 commits for this small change

Appreciate the fix though! I do agree using a negative on a sup def is a bit weird

**Review by mikinty — CHANGES_REQUESTED** [review]
Squash commits into 1

File: chapters/chapter1/chapter1-3.tex (hunk 1 above)
Comment (LaMavia) [reply chapters/chapter1/chapter1-3.tex]: Sure thing

**Comment by DerivedMate** [comment]
It appears to have included one of the previous commits (sorry, it was my first time squashing commits, and I've just woken up) but it should be all fine now

**Comment by LaMavia** [comment]
It appears to have included 2 previous commits (sorry for that; it was my first time squashing, and I've only just woken up), but it should be more or less fine

**Review by mikinty — CHANGES_REQUESTED** [review]
Hmm looks like you also committed some extra files (README, pdf, 8-4) -- do you mind only committing the chapter1-3 change?

**Comment by LaMavia** [comment]
Sorry it took so long; as you can tell, I'm not particularly fluent in git

**Comment by mikinty** [comment]
> Sorry it took so long; as you can tell, I'm not particularly fluent in git

No worry, thanks for working this through with me.

Git is a learning process and has a really steep learning curve and really particular issues. Hopefully you learned some new skills along the way.

Thanks for your change! I'll build this and merge to master

**Review by mikinty — APPROVED** [review]

**Pull request merged**